hydrolib/core/io/dflowfm/ini/serializer.py

```diff
@@ -57,6 +57,35 @@ def total_datablock_indent(self) -> int:
         return self.section_indent + self.datablock_indent
 
 
+class DataBlockINIBasedSerializerConfig(INISerializerConfig):
```

Comment:
I agree with this comment. 
However, this would require a bigger refactoring.
The SectionSerializer and the Serializer both use the INISerializerConfig and use its data block settings. To refactor this we should have a Serializer for section with data blocks. 
So unfortunately, I will skip this for now and create a new issue.

Reply:
Created issue: #424